dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/JarLinkageReport.java

```diff
@@ -34,6 +34,9 @@
 
   public abstract ImmutableList<StaticLinkageError<MethodSymbolReference>> getMissingMethodErrors();
 
+  public abstract ImmutableList<StaticLinkageError<MethodSymbolReference>>
+      getMissingInterfaceMethodErrors();
```

Comment:
If `MethodSymbolReference` itself encoded whether it was an interface method, would you still need this separate property?

Reply:
No, it's not needed. Let me implement in that way...